lib/linter/linter.js

```diff
@@ -800,13 +800,15 @@ function parse(text, languageOptions, filePath) {
      * problem that ESLint identified just like any other.
      */
     try {
+        debug("Parsing:", filePath);
         const parseResult = (typeof parser.parseForESLint === "function")
             ? parser.parseForESLint(textToParse, parserOptions)
             : { ast: parser.parse(textToParse, parserOptions) };
         const ast = parseResult.ast;
         const parserServices = parseResult.services || {};
         const visitorKeys = parseResult.visitorKeys || evk.KEYS;
         const scopeManager = parseResult.scopeManager || analyzeScope(ast, languageOptions, visitorKeys);
+        debug("Parsing successful:", filePath);
```

Comment:
```suggestion
        const scopeManager = parseResult.scopeManager || analyzeScope(ast, languageOptions, visitorKeys);

        debug("Parsing successful:", filePath);
```

This should fix the lint error.

Reply:
Thanks!